### PR TITLE
Create Postman environment and collection.

### PR DESCRIPTION
**Pull request by chrismiddleton-latch**

### CODE REVIEW

File: util/postman/openkit.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,289 @@
+{
```
Comment (usmanaz) [inline util/postman/openkit.postman_collection.json:1]: 🔥🔥🔥! Out of curiosity, did you generate the same way we exported postman collections in the past?

File: util/postman/openkit.postman_environment.json [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+			"value": "",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "param_end_time",
+			"value": "",
+			"type": "default",
+			"enabled": true
+		}
```
Comment (usmanaz) [inline util/postman/openkit.postman_environment.json:118]: Do we want to include the `passcodeType` param as well here?

File: util/postman/openkit.postman_environment.json [hunk 3]
```diff
@@ -0,0 +1,123 @@
+{
+	"id": "b4c4d4a9-7480-4823-98a2-60bdf5b7f4fe",
+	"name": "openkit - prod",
```
Comment (usmanaz) [inline util/postman/openkit.postman_environment.json:3]: nit: Is this to distinguish between partner's dev and prod environments (like for Landing)? Could we just leave it as `openkit`?

Comment (chrismiddleton-latch) [reply util/postman/openkit.postman_environment.json:3]: Agreed, will keep it as `openkit` for now. In the future we will support multiple environments (hence the URLs will be different), but agree that for now we can simplify it.

File: util/postman/openkit.postman_environment.json (hunk 2 above)
Comment (chrismiddleton-latch) [reply util/postman/openkit.postman_environment.json:118]: The current APIs only support `PERMANENT` passcode, so it's hard-coded in the request. Once we expose the other endpoints, we can add it.

File: util/postman/openkit.postman_collection.json (hunk 1 above)
Comment (chrismiddleton-latch) [reply util/postman/openkit.postman_collection.json:1]: Correct. The exact same way.

**Pull request merged**